client/src/App.jsx

```diff
@@ -1,20 +1,19 @@
 import React from "react";
 import { MuiThemeProvider } from "@material-ui/core";
-import { BrowserRouter, Route } from "react-router-dom";
-
+import { BrowserRouter, Route, Redirect } from "react-router-dom";
 import theme from "./themes/theme";
-import LandingPage from "./pages/Landing";
-
-import TestLanding from "./pages/TestLanding.js";
+import SignUp from "./components/Sign-up";
```

Comment:
I feel like component filenames should also be PascalCase (so SignUp vs Sign-up)